plugins/Dispersion/DispersionControlDialog.cpp

```diff
@@ -24,12 +24,14 @@
 
 
 #include "DispersionControlDialog.h"
+#include <QBoxLayout>
+#include <qnamespace.h>
```

Comment:
You should be able to write `#include <QNamespace>` (or maybe its named something different?). That's usually how we would include these files.

Reply:
Good catch 👍 will do